apps/prairielearn/src/tests/questionSharing.test.ts

```diff
@@ -103,62 +163,66 @@ describe('Question Sharing', function () {
+        assert(!(await res.text()).includes(SHARING_QUESTION_QID));
 
         // Question can be accessed through the owning course
         const questionId = (
           await sqldb.queryOneRowAsync(sql.get_question_id, {
-            course_id: exampleCourseId,
-            qid: sharedQuestionQid,
+            course_id: sharingCourse.id,
+            qid: SHARING_QUESTION_QID,
           })
         ).rows[0].id;
-        const sharedQuestionUrl = `${baseUrl}/course_instance/${exampleCourseId}/instructor/question/${questionId}`;
+        const sharedQuestionUrl = `${baseUrl}/course_instance/${sharingCourse.id}/instructor/question/${questionId}`;
         const sharedQuestionPage = await fetchCheerio(sharedQuestionUrl);
         assert(sharedQuestionPage.ok);
 
         // Question cannot be accessed through the consuming course, sharing permissions not yet set
-        const sharedQuestionSharedUrl = `${baseUrl}/course_instance/${testCourseId}/instructor/question/${questionId}/settings`;
+        const sharedQuestionSharedUrl = `${baseUrl}/course_instance/${consumingCourse.id}/instructor/question/${questionId}/settings`;
         const sharedQuestionSharedPage = await fetchCheerio(sharedQuestionSharedUrl);
         assert(!sharedQuestionSharedPage.ok);
       },
     );
 
     step('Fail if trying to set an invalid sharing name', async () => {
-      let res = await setSharingName(testCourseId, 'invalid@sharingname');
+      let res = await setSharingName(sharingCourse.id, 'invalid@sharingname');
       assert(res.status === 400);
 
-      res = await setSharingName(testCourseId, 'invalid / sharingname');
+      res = await setSharingName(sharingCourse.id, 'invalid / sharingname');
       assert(res.status === 400);
 
-      res = await setSharingName(testCourseId, '');
+      res = await setSharingName(sharingCourse.id, '');
       assert(res.status === 400);
     });
 
-    step('Set test course sharing name', async () => {
-      await setSharingName(testCourseId, testCourseSharingName);
-      const sharingPage = await fetchCheerio(sharingPageUrl(testCourseId));
+    step('Set consuming course sharing name', async () => {
+      await setSharingName(consumingCourse.id, CONSUMING_COURSE_SHARING_NAME);
+      const sharingPage = await fetchCheerio(sharingPageUrl(consumingCourse.id));
       assert(sharingPage.ok);
-      assert.include(sharingPage.$('[data-testid="sharing-name"]').text(), testCourseSharingName);
+      assert.include(
+        sharingPage.$('[data-testid="sharing-name"]').text(),
+        CONSUMING_COURSE_SHARING_NAME,
+      );
     });
 
+    // TODO: fix this test?
     step('Fail if trying to set sharing name again.', async () => {
-      const result = await setSharingName(testCourseId, testCourseSharingName);
+      const result = await setSharingName(consumingCourse.id, CONSUMING_COURSE_SHARING_NAME);
       assert.equal(result.status, 200);
     });
```

Comment:
Can you be more specific in your comment? what about this test do you want to fix?

Reply:
The test is named "Fail...", but it asserts a 200 response. Either the title or the assertion (or the behavior?) is wrong.